pdc/apps/utils/utils.py

```diff
@@ -59,3 +61,43 @@ def read_permission_for_all():
 
 def get_model_name_from_obj_or_cls(obj_or_cls):
     return ContentType.objects.get_for_model(obj_or_cls).model
+
+
+class RelatedNestedOrderingFilter(OrderingFilter):
+    """
+    Extends OrderingFilter to support ordering by fields in related models
+    using Django orm '__'.
+
+    This class is copied and changed from the patch of the github issue.
+    https://github.com/encode/django-rest-framework/issues/1005
```

Comment:
Well the latest comment takes into account "reverse relation". But ignore it if it won't work for with the version of rest-framework we use.

Since there is lot of comments with code in the issue referenced by the docstring, a link to exact comment with the code your change was based on would help a bit more.